src/main.js

```diff
@@ -1,23 +1,21 @@
-import image from "../templates/favorite-icon-filled.png";
-
-console.log("npm run dev 명령어를 통해 점심 뭐 먹지 미션을 시작하세요");
-console.log(
-  "%c ___       ___  ___  ________   ________  ___  ___     \n" +
-    "|\\  \\     |\\  \\|\\  \\|\\   ___  \\|\\   ____\\|\\  \\|\\  \\    \n" +
-    "\\ \\  \\    \\ \\  \\\\\\  \\ \\  \\\\ \\  \\ \\  \\___|\\ \\  \\\\\\  \\   \n" +
-    " \\ \\  \\    \\ \\  \\\\\\  \\ \\  \\\\ \\  \\ \\  \\    \\ \\   __  \\  \n" +
-    "  \\ \\  \\____\\ \\  \\\\\\  \\ \\  \\\\ \\  \\ \\  \\____\\ \\  \\ \\  \\ \n" +
-    "   \\ \\_______\\ \\_______\\ \\__\\\\ \\__\\ \\_______\\ \\__\\ \\__\\\n" +
-    "    \\|_______|\\|_______|\\|__| \\|__|\\|_______|\\|__|\\|__|",
-  "color: #d81b60; font-size: 14px; font-weight: bold;"
-);
+import { $ } from "./utils/dom";
+import header from "./components/header";
+import Modal from "./components/common/modal";
+import Title from "./components/common/title";
+import RegisterForm from "./components/registerForm";
+import renderRestaurants from "./domain/renderRestaurant";
+import { registerModalClose } from "./components/common/modal/handleCloseModal";
+import { restaurantList } from "./restaurantList";
 
 addEventListener("load", () => {
-  const app = document.querySelector("#app");
-  const buttonImage = document.createElement("img");
-  buttonImage.src = image;
+  $("#app").prepend(header());
+  renderRestaurants(...restaurantList);
 
-  if (app) {
-    app.appendChild(buttonImage);
-  }
+  $("main").appendChild(
+    Modal(
+      registerModalClose,
+      Title("새로운 음식점", "h2", "modal-title", "text-title"),
+      RegisterForm()
+    )
```

Comment:
컴포넌트가 상위 컴포넌트에서 핸들러나 다른 컴포넌트를 파라미터로 받게 하는 것은 재사용성을 높여주는 아주 좋은 패턴입니다. 다만 여기서 매개변수를 객체 형식으로 넘겨주면, 다른 개발자가 보기에 각각이 무엇을 의미하는지 더 직관적으로 파악될 수 있을 것 같습니다.

```suggestion
    Modal({
      close: registerModalClose,
      headerComponent: Title("새로운 음식점", "h2", "modal-title", "text-title"),
      bodyComponent: RegisterForm()
    })
```

Reply:
https://github.com/woowacourse/javascript-lunch/pull/197/commits/47bf288526934434d16a2f328c95370c20daccad 반영했습니다!